packages/contact-center/store/package.json

```diff
@@ -18,11 +18,11 @@
     "test:unit": "jest"
   },
   "dependencies": {
+    "@webex/plugin-cc": "3.5.0-wxcc.6",
```

Comment:
Using only @webex/plugin-cc rather than the whole webex object brought down the bundle size of store from 17MB to 11 MB.

Reply:
After making this changes webex is not getting initialised properly and we get an error this.cc is undefined. Since we already have a ticket for optimizing the bundle sizes I will add this as part of thaat ticket as well.